src/features/metadata-based-view/MetadataBasedItemList.js

```diff
@@ -285,7 +159,13 @@ class MetadataBasedItemList extends React.Component<Props, State> {
         });
 
         return (
-            <div key={key} className={classes} style={style}>
+            <div
+                key={key}
+                className={classes}
+                style={style}
+                onMouseLeave={this.handleMouseLeave}
+                onMouseEnter={() => this.handleMouseEnter(columnIndex, rowIndex)}
```

Comment:
Minor: can we alphabetize these attributes?

Reply:
@sahiga done!